# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../fast-user-switch-applet/compile-fix.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -NrU5 fast-user-switch-applet-2.24.0.orig/configure.ac fast-user-switch-applet-2.24.0/configure.ac
--- fast-user-switch-applet-2.24.0.orig/configure.ac	2009-05-26 18:19:14.000000000 +0200
+++ fast-user-switch-applet-2.24.0/configure.ac	2009-05-26 18:22:55.000000000 +0200
@@ -49,11 +49,12 @@
 GLIB_REQUIRED_VERSION=2.15.3
 
 PKG_CHECK_MODULES(APPLET, gtk+-2.0 >= $GTK_REQUIRED_VERSION
 	     libpanelapplet-2.0 >= $PANEL_REQUIRED_VERSION
 	     libglade-2.0 >= $GLADE_REQUIRED_VERSION
-	     gio-2.0 >= $GLIB_REQUIRED_VERSION)
+	     gio-2.0 >= $GLIB_REQUIRED_VERSION
+	     libgnomeui-2.0)
 AC_SUBST(APPLET_CFLAGS)
 AC_SUBST(APPLET_LIBS)
 
 AC_CHECK_FUNCS([setresuid setenv unsetenv clearenv])
 
